cmd/thv-operator/controllers/mcpserver_controller.go

```diff
@@ -136,7 +136,7 @@ func (r *MCPServerReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	// Check if the Service already exists, if not create a new one
 	service := &corev1.Service{}
 	err = r.Get(ctx, types.NamespacedName{Name: mcpServer.Name, Namespace: mcpServer.Namespace}, service)
-	if err != nil && errors.IsNotFound(err) {
```

Comment:
is it so that we shouldn't deploy it? This is the service for the proxy which we should deploy. I think we should just rename it instead.

Reply:
repushed for the rename